app/retail/views.py

```diff
@@ -54,6 +54,9 @@ def index(request):
         'slides': slides,
         'slideDurationInMs': 6000,
         'active': 'home',
+        'hide_newsletter_caption': True,
```

Comment:
What's the utility of this bool? Are we altering this value at any point? 🤔

Reply:
@mbeacom I think what @thelostone-mc has done here is he is sending this boolean only to landing-page, which makes it non-hidden in all the other pages. Hence both the caption and consent are shown in every other page except the landing page.